articles/blockchain-as-a-revolution-inhealth/index.md

```diff
@@ -0,0 +1,86 @@
+>**Blockchain has the potential to keep private information safe and secure while still being accessible anywhere.**
+
+***Medical Device Usage Tracking***
+The use of  medical devices such as electronic wearables have been on the rise, with new reports revealing that forty percent of medical patients use such devices to track their health. However, this makes it difficult to store all the  daily medical reports from devicse for each patients. This is another area where keeping blockchain records can help.
+
+***Drug Supply Chain Tracking***
+According to Hudson River Field Office (HRFO), over fifteen percent of drugs sold in developing countries are counterfeit. In addition, pharmaceutical companies lose about $200 billion annually due to counterfeiting. Blockchain can curb this trend with features like point-by-point, which tracks possibilities and proof of authenticity. This allows users to verify drug authenticity before purchase or consumption.
+
+### Digital Identity
+***Patient Identity Validation***
+Blockchain can solve the issue of patient identity theft and can reduce the use of fake IDs by impersonators.
+
+***Healthcare Provider Identity Validation***
+Impersonation using fake medical records is a common problem in third world countries and has caused great number of deaths. Blockchain can address these issues with its validation system.
+
+***Drug Identity Validation***
+Drug identity can be validated using blockchain technology. Having transparent information about particular drugs can reduce the amount of fake drugs in circulation.
+
+### Financial Transactions
+***Direct Healthcare Service Payment***
+Blockchain technology has been used extensively for financial transactions, with the most famous use case being Bitcoin, and this model can also be transferred to healthcare.
+
+### Smart Contracts
+***Backing Research & Treatment of Healthcare Data***
+There have been several challenges when coordinating research and clinical trials since it requires a collaborative effort across many institutions. This can lead to theft of intellectual property and dishonesty. Using blockchain can change this by enabling users to authenticate their documents, ensuring the proof of existence. Test results, statistics, and quality reports can be shared amongst researchers without any fear of intellectual property theft.
+
+#### Implementation Challenges and Considerations
+Although Blockchain technology is not fully mature today, it presents numerous opportunities for healthcare. Some features of can already be implemented today, but before blockchain can be adopted by organizations nationwide, it must evolve to address potential behavioral, technical, organizational, and economical considerations.
```

Comment:
You mention that there is work to be done across several areas. Give an overview of what these future areas for improvement might be and why they aren't available yet.